From e1a8de26ea1ba0b1677d6e48c2417b9ad4ccb3ec Mon Sep 17 00:00:00 2001
From: Ronin <58588775+gitr0n1n@users.noreply.github.com>
Date: Thu, 3 Dec 2020 16:54:40 -0700
Subject: [PATCH 1/2] Update ecip-1049.md

+ move to Draft Status (Expired Last Call Review on Oct 9, 2020) due to material redraft of proposal.
+ Update Discussion Thread to new Redraft thread
+ Related Discussions Section
---
 _specs/ecip-1049.md | 28 +++++++++++++++++++---------
 1 file changed, 19 insertions(+), 9 deletions(-)

diff --git a/_specs/ecip-1049.md b/_specs/ecip-1049.md
index 354ded306..7a519ed39 100644
--- a/_specs/ecip-1049.md
+++ b/_specs/ecip-1049.md
@@ -2,11 +2,10 @@
 lang: en
 ecip: 1049
 title: Change the ETC Proof of Work Algorithm to Keccak256
-status: Last Call
-review-period-end: 2020-10-09
+status: Draft
 type: Standards Track
 category: Core
-discussions-to: https://github.com/ethereumclassic/ECIPs/issues/13
+discussions-to: https://github.com/ethereumclassic/ECIPs/issues/394
 author: Alexander Tsankov <alexander.tsankov@colorado.edu>
 created: 2019-01-08
 ---
@@ -56,7 +55,7 @@ Here is an analysis of Monero's nonce-distribution for "cryptonight", an algorit
 Based on analysis of the EVM architecture [here](https://cdn.discordapp.com/attachments/223675625334898688/534597157693685760/eth.jpg) there are two main pieces that need to be changed:
 
 1. The Proof of work function needs to be replaced with Keccak256
-1. The Function that checks the nonce-header  in the block needs to know to accept Keccak256 hashes as valid for a block.
+2. The Function that checks the nonce-header  in the block needs to know to accept Keccak256 hashes as valid for a block.
 
 ![example](https://i.imgur.com/2hobqOL.png)
 
@@ -67,8 +66,19 @@ The area where most of the changes will be in `multi-geth/consensus`
 ### References: 
 
 1. https://github.com/ethereum/wiki/wiki/Dagger-Hashimoto#introduction
-1. https://en.wikipedia.org/wiki/SHA-3
-1. https://satoshi.nakamotoinstitute.org/emails/cryptography/2/
-1. https://github.com/0xbitcoin/white-paper
-1. https://github.com/EthereumCommonwealth/Auditing/issues/102
-1. https://etherscan.io/address/0xb6ed7644c69416d67b522e20bc294a9a9b405b31
+2. https://en.wikipedia.org/wiki/SHA-3
+3. https://satoshi.nakamotoinstitute.org/emails/cryptography/2/
+4. https://github.com/0xbitcoin/white-paper
+5. https://github.com/EthereumCommonwealth/Auditing/issues/102
+6. https://etherscan.io/address/0xb6ed7644c69416d67b522e20bc294a9a9b405b31
+
+### Related Discussions: 
+
+1. https://github.com/ethereumclassic/ECIPs/pull/8
+2. https://github.com/ethereumclassic/ECIPs/issues/13
+3. https://github.com/ethereumclassic/ECIPs/issues/342
+4. https://github.com/ethereumclassic/ECIPs/issues/333
+5. https://github.com/ethereumclassic/ECIPs/issues/362
+6. https://github.com/ethereumclassic/ECIPs/issues/382
+7. https://github.com/ethereum/EIPs/issues/2951
+8. https://github.com/ethereumclassic/ECIPs/issues/394

From 5ae7308b7b3d4c80674b3c9d6d9c92d9cf43ffe2 Mon Sep 17 00:00:00 2001
From: Ronin <58588775+gitr0n1n@users.noreply.github.com>
Date: Thu, 3 Dec 2020 19:38:56 -0700
Subject: [PATCH 2/2] Update ecip-1049.md

ECIP Process Formality:
+ Move to Rejected after failing in Last Call by Ot 9, 2020.
---
 _specs/ecip-1049.md | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/_specs/ecip-1049.md b/_specs/ecip-1049.md
index 7a519ed39..4ef0a1bca 100644
--- a/_specs/ecip-1049.md
+++ b/_specs/ecip-1049.md
@@ -2,7 +2,7 @@
 lang: en
 ecip: 1049
 title: Change the ETC Proof of Work Algorithm to Keccak256
-status: Draft
+status: Rejected
 type: Standards Track
 category: Core
 discussions-to: https://github.com/ethereumclassic/ECIPs/issues/394
